Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -323,7 +323,11 @@ private void InitializeServiceLocator()
             //If the Mixed Reality Toolkit is not configured, stop.
             if (ActiveProfile == null)
             {
-                Debug.LogError("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
+                #if UNITY_EDITOR
```

Comment:
Use [`Application.isEditor`](https://docs.unity3d.com/ScriptReference/Application-isEditor.html) here? Since these aren't editor-specific APIs, just editor-specific behavior.

Reply:
Going to combine with Application.isPlaying. I want to exclude just edit mode